services/web/pkg/config/options.go

```diff
@@ -0,0 +1,65 @@
+	DisablePreviews          bool            `json:"disablePreviews" yaml:"disablePreviews" env:"WEB_OPTION_DISABLE_PREVIEWS" desc:"Set this option to 'true' to disable previews in all the different file listing views. The only list view that is not affected by this is the trash bin, as that doesn't allow showing previews at all."`
+	PreviewFileMimeTypes     []string        `json:"previewFileMimeTypes" yaml:"previewFileMimeTypes" env:"WEB_OPTION_PREVIEW_FILE_MIMETYPES" desc:"Specifies which mimeTypes will be previewed in the ui. For example to only preview jpg and text files set this option to ['image/jpeg', 'text/plain']."`
+	AccountEditLink          AccountEditLink `json:"accountEditLink" yaml:"accountEditLink"`
+	DisableFeedbackLink      bool            `json:"disableFeedbackLink" yaml:"disableFeedbackLink" env:"WEB_OPTION_DISABLE_FEEDBACK_LINK" desc:"Set this option to 'true' to disable the feedback link in the topbar. Keeping it enabled (value 'false' or absence of the option) allows ownCloud to get feedback from your user base through a dedicated survey website."`
+	FeedbackLink             FeedbackLink    `json:"feedbackLink" yaml:"feedbackLink"`
+	SharingRecipientsPerPage int             `json:"sharingRecipientsPerPage" yaml:"sharingRecipientsPerPage" env:"WEB_OPTION_SHARING_RECIPIENTS_PER_PAGE" desc:"Sets the amount of users shown as recipients in the dropdown when sharing resources. Default amount is 200."`
+	Sidebar                  Sidebar         `json:"sidebar" yaml:"sidebar"`
+	RunningOnEOS             bool            `json:"runningOnEos" yaml:"runningOnEos" env:"WEB_OPTION_RUNNING_ON_EOS" desc:"Set this option to 'true' if running on an [EOS storage backend](https://eos-web.web.cern.ch/eos-web/) to enable its specific features. Defaults to 'false'."`
+	CernFeatures             bool            `json:"cernFeatures" yaml:"cernFeatures" env:"WEB_OPTION_CERN_FEATURES" desc:"Enabling this will activate CERN-specific features. Defaults to 'false'."`
+	HoverableQuickActions    bool            `json:"hoverableQuickActions" yaml:"hoverableQuickActions" env:"WEB_OPTION_HOVERABLE_QUICK_ACTIONS" desc:"Set this option to 'true' to hide the quick actions (buttons appearing on file rows), and only show them when the user hovers the row with his mouse. Defaults to 'false'."`
+	Routing                  Routing         `json:"routing" yaml:"routing"`
+	Upload                   Upload          `json:"upload" yaml:"upload"`
+	Editor                   Editor          `json:"editor" yaml:"editor"`
+	ContextHelpersReadMore   bool            `json:"contextHelpersReadMore" yaml:"contextHelpersReadMore" env:"WEB_OPTION_CONTEXTHELPERS_READ_MORE" desc:"Specifies whether the 'Read more' link should be displayed or not."`
+	LogoutURL                string          `json:"logoutURL" yaml:"logoutURL" env:"WEB_OPTION_LOGOUT_URL" desc:"The logout url"`
```

Comment:
```suggestion
	LogoutURL                string          `json:"logoutURL" yaml:"logoutURL" env:"WEB_OPTION_LOGOUT_URL" desc:"Adds a link to the user's profile page to point him to an external page, where he can manage his session and devices. This is helpful when an external IdP is used. This option is disabled by default."`
```

Reply:
Please use
- `to point them` instead of `to point him`
- `where they can manage their session and devices` instead of `where he can manage his session and devices`
in the proposed text (them/they is the correct gender neutral singular form).